thirty-three-across/thirty-three-across-htb.js

```diff
@@ -438,11 +526,11 @@ function ThirtyThreeAcrossHtb(configs) {
             },
 
```

Comment:
What is `demandExpiry` from https://github.com/33Across/ix-library-adapters/blob/8e8e772e7a59812b64ab20d9c6b92fcaeeb562f9/thirty-three-across/thirty-three-across-htb.js#L511

Reply:
Extra features that a adapter can support
demandExpiry - Setting an expiry time on the demand that adapter returns.

Reply:
Like bid TTL? In which case can we make this to be 60s to match prebid?